docs/installation.rst

```diff
@@ -36,73 +32,83 @@ using
 
     C:\> conda install -c conda-forge openjpeg
 
-3. Install
-TIAToolbox.
-
-.. code-block:: console
-
-    C:\> pip install tiatoolbox
-
-Linux (Ubuntu)
-^^^^^^^^^^^^^^
-On Linux the prerequisite software can be installed using the command
-
-.. code-block:: console
-
-    $ apt-get -y install libopenjp2-7-dev libopenjp2-tools openslide-tools
-
-The same command is used when working on the Colab or Kaggle platforms.
-When working on Google Colab, we remove the packages ``datascience`` and ``albumentations`` because they conflict
-and produce an error message.
-
 macOS
-^^^^^
+-----
 
 On macOS there are two popular package managers, `homebrew`_ and `macports`_.
 
 .. _homebrew: https://brew.sh/
 .. _macports: https://www.macports.org/
 
 Homebrew
-""""""""
+^^^^^^^^
 
 .. code-block:: console
 
     $ brew install openjpeg openslide
 
 MacPorts
-""""""""
+^^^^^^^^
 
 .. code-block:: console
 
     $ port install openjpeg openslide
 
 
-Stable release
---------------
+Installing Stable Release (Recommended)
+=======================================
 
 Please note that TIAToolbox is tested for python version 3.8, 3.9 and 3.10.
-To install TIA Toolbox, run this command in your terminal after you have installed the prerequisite software:
+
+New
```

Comment:
I don't think having a subheader of "New" is required here. Just have subheaders for upgrades and without dependencies.

Reply:
ok let me change it to Recommended and remove recommended from the title.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/d221593ae51a15d2743d3b60a0bcb2ceee7344fb